README.md

```diff
@@ -118,7 +118,7 @@ spec:
         driver: csi.cert-manager.io
         volumeAttributes:
               csi.cert-manager.io/issuer-name: ca-issuer
-              csi.cert-manager.io/dns-names: my-service.sandbox.svc.cluster.local
+              csi.cert-manager.io/dns-names: {{.POD_NAME}}.{{.POD_NAMESPACE}}.svc.cluster.local
```

Comment:
is this still correct after the os.Expand change?

Reply:
I guess the semantics are a bit different now since this is now the pod name, rather than the arbitrary hard coded service `my-service`. Do you think it is worth changing back?